elasticsearch_dsl/faceted_search_base.py

```diff
@@ -361,10 +364,7 @@ def __init__(
 
         self._s = self.build_search()
 
-    def count(self) -> int:
-        return self._s.count()
```

Comment:
Why remove `count()`? This sounds like it could be useful somehow

Reply:
Because it is in the wrong place. This is something I missed when we did the async changes, `count()` should be a coroutine in the async case, so it has to move down to the async/sync subclasses of `SearchBase`. Look in `_async/faceted_search.py` and `_sync/faceted_search.py` for it.